### PR TITLE
[13.0][IMP] stock_report_quantity_by_location: show qty on hand, reserved and unreserved

### PR DESCRIPTION
**Pull request by JordiMForgeFlow**
Add the on hand, reserved and unreserved qty in the report so that users can see them all in the same report at the same time.

### CODE REVIEW

File: stock_report_quantity_by_location/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Stock Report Quantity By Location",
     "summary": "Stock Report Quantity By Location",
-    "version": "13.0.1.1.0",
+    "version": "13.0.2.1.0",
```
Comment (MiquelRForgeFlow) [inline stock_report_quantity_by_location/__manifest__.py]: don't, this will be done by the bot

Comment (JordiMForgeFlow) [reply stock_report_quantity_by_location/__manifest__.py]: True 🤦🏼‍♂️

**Review by GuillemCForgeFlow — APPROVED** [review]
LGTM! Functional review :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
Code review LG!

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**